docs/_basic/listening_modals.md

```diff
@@ -69,3 +71,23 @@ app.view('view_b', async ({ ack, body, view, client }) => {
 
 });
 ```
+
+---
+
+##### handle view closed events
+
+The built-in `view()` method can also handle `view_closed` events, which are sent when a <a href="https://api.slack.com/reference/block-kit/views">view payload</a> has the `notify_on_close` flag set.
+The event contains the respective `callback_id` of the closed view.
+Please note that you have to specify the `type: 'view_closed'` in the constraint object explicitly, since otherwise the handler is not triggered.
```

Comment:
```suggestion
💡 When listening for `view_closed` requests, you must pass an object containing `type: 'view_closed'` and the view `callback_id`. See below for an example of this:
```

Reply:
While it's recommended to have `type` if your app handles `view_closed`, if an `app.view` listener does not have `type`, the listener handles both `view_submission` and `view_closed`. Thus, "you must pass an object containing `type`" is not accurate here.

Reply:
Ok I just double checked and at least for me this is not working as described by you @seratch, which is the reason why I decided to document it. Unfortunatly even on log level DEBUG I cannot see a reason why the event is not handled at all but when the listener does not have the type property set in the end all close events result in a 404 for me.

Might by that this is not the intended behavior and I found a bug :shrug: I will try to provide a test which documents my observation.

Reply:
@TheManWhoStaresAtCode I've checked this again and you're right here. Never mind!

Reply:
ok I personally would favor the suggested behavior but probably that is another topic and not part of this ticket, right?

Reply:
ok I personally would favor the suggested behavior but probably that is another topic and not part of this ticket, right?